client/rpc/src/eth.rs

```diff
@@ -603,7 +603,7 @@ where
 			Some(number),
 		)? {
 			Some(id) => id,
-			None => return Ok(None),
+			None => U256::from(339330),
```

Comment:
This looks like a hack, what’s about to make it optional? How it will works on non-Shiden chains?